test/src/tests-utils.ts

```diff
@@ -330,4 +330,11 @@ describe('Utils', () => {
             expect(isEmpty(undefined)).to.equal(true);
         });
     });
+
+    describe('hashing', () => {
+        it('hashing utility class should have certain functions on it', () => {
```

Comment:
Is this something you for a different PR or did is this still a work in progress?